src/bench/wallet_loading.cpp

```diff
@@ -47,12 +47,11 @@ static void BenchUnloadWallet(std::shared_ptr<CWallet>&& wallet)
 
 static void AddTx(CWallet& wallet)
 {
-    bilingual_str error;
-    CTxDestination dest;
-    wallet.GetNewDestination(OutputType::BECH32, "", dest, error);
+    const auto& dest = wallet.GetNewDestination(OutputType::BECH32, "");
+    assert(dest.HasRes());
```

Comment:
nit: Could write shorter as `const CTxDestination& dest{Assert(wallet.GetNewDestination(OutputType::BECH32, "")).GetObj()};`

Reply:
Actually nvm. This would likely be a use-after-free.

Reply:
Maybe there could be a `ReleaseObj` helper to obtain the ownership?